commands/snapshotdeletecommand.php

```diff
@@ -0,0 +1,143 @@
+        }
+        else throw new Exception("Error during reading config.ini!");
+
+        $profitBricksApi = new ProfitBricksApi();
+        $profitBricksApi->setUserName($this->config["api"]["user"]);
+        $profitBricksApi->setPassword($this->config["api"]["password"]);
+
+        $snapshots = $profitBricksApi->snapshots();
+
+        $io =  new SymfonyStyle($input, $output);
+        $tableRows = array ();
+        $header = array ("ID", "Name", "Description", "Date       Time", "Size", "State");
+        $sumSize = 0;
+        $sumCount = 0;
+
+        if (count($input->getArgument("snapshotId"))>0)
+        {
+            $io->title("Snapshots Deletion");
+            foreach ($input->getArgument("snapshotId") as $id)
+            {
+                if (isset($snapshots[$id]) && $snapshots[$id]->createdDate<$this->before)
+                {
+                    if ($profitBricksApi->deleteSnapshot($snapshots[$id]->id))
+                    {
+                        $tableRows[] = array ($id, $snapshots[$id]->name, $snapshots[$id]->description, $snapshots[$id]->createdDate->format("d.m.Y H:i"), $snapshots[$id]->size." GB", "deleted!");
+                        $sumSize += (int)$snapshots[$id]->size;
+                        $sumCount += 1;
+                    }
+                    else $tableRows[] = array ($id, $snapshots[$id]->name, $snapshots[$id]->description, $snapshots[$id]->createdDate->format("d.m.Y H:i"), $snapshots[$id]->size." GB", "deletion failed!");
+                }
+            }
+        }
+        else
+        {
+            if ($input->getOption("quiet")=== false)
+            {// In quiet mode don't ask for continuing this action.
+                $output->writeln("This will delete every snapshot!");
+                $helper = $this->getHelper("question");
+                $question = new ConfirmationQuestion("Continue with this action?", false);
+                if (!$helper->ask($input, $output, $question)) return;
+            }
+            $io->title("Snapshots Deletion");
+            foreach ($snapshots as $snapshot)
+            {
+                if ($snapshot->createdDate<$this->before)
+                {
+                    if ($profitBricksApi->deleteSnapshot($snapshot->id))
+                    {
+                        $tableRows[] = array ($snapshot->id, $snapshot->name, $snapshot->description, $snapshot->createdDate->format("d.m.Y H:i"), $snapshot->size." GB", "deleted!");
+                        $sumSize += (int)$snapshot->size;
+                        $sumCount += 1;
+                    }
+                    else $tableRows[] = array ($snapshot->id, $snapshot->name, $snapshot->description, $snapshot->createdDate->format("d.m.Y H:i"), $snapshot->size." GB", "deletion failed!");
+                }
+            }
+        }
+        $sumSize = ceil($sumSize / 100);
+        $sumSize = $sumSize / 10;
+        $sumSize = str_replace('.',',',(string)$sumSize);
```

Comment:
It is quite common to have a method like niceSize(int $_size) which returns a fully divided and human readable string.
If it comes to values from kilobytes to terabytes with multiple transformations, this would be needed here.